src/System.Management.Automation/utils/FuzzyMatch.cs

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// Based off https://www.csharpstar.com/csharp-string-distance-algorithm/
+
+using System;
+using System.Collections.Generic;
+
+namespace System.Management.Automation
+{
+    internal static class FuzzyMatcher
+    {
+        public const int MinimumDistance = 5;
+
+        /// <summary>
+        /// Determine if the two strings are considered similar.
+        /// </summary>
+        /// <param name="string1">The first string to compare.</param>
+        /// <param name="string2">The second string to compare.</param>
+        /// <returns>True if the two strings have a distance <= MinimumDistance.</returns>
+        public static bool IsFuzzyMatch(string string1, string string2)
+        {
+            return GetDamerauLevenshteinDistance(string1, string2) <= MinimumDistance;
+        }
+
+
+        /// <summary>
+        /// Compute the distance between two strings.
```

Comment:
Worth adding info as to where this implementation comes from

Reply:
It's in the comment at the top of the file.  I'll move it to the summary here.